ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.9__4.5.yaml

```diff
@@ -25,9 +25,6 @@ images:
 - context_dir: knative-operator
   from: base
   to: knative-operator
-- context_dir: olm-catalog/serverless-operator
-  from: base
-  to: serverless-bundle
```

Comment:
I am now based on "master" biuts, but ... this OLM stuff is now out (compared to 1.8)

Reply:
Oh. Yeah. That's why I said "combined 1.8 with master" on Slack. We need to put it here as well - for future releases.

Reply:
It will be used here in future releases (1.10): https://github.com/openshift-knative/serverless-operator/blob/release-1.9/hack/lib/catalogsource.bash#L83